Cargo.toml

```diff
@@ -53,6 +53,7 @@ reqwest = { version = "0.10", features = ["blocking", "json"] }
 log = { version = "0.4", features = ["kv_unstable"] }
 insta = "1.0"
 env_logger = "0.8"
+serde_qs_test = { package = "serde_qs", version = "0.8.1", features = ["actix"] }
```

Comment:
The version of `serde_qs` in the core is `0`. It means at some point in time, tests may pass, but the code can be broken.